src/App.js

```diff
@@ -32,7 +32,7 @@ function App() {
     return isTokenValid(token).then((listExists) => {
       if (listExists) {
         // if isTokenValid returns true the token is taken, so try again
-        createList();
+        return createList();
```

Comment:
Curious as to why we change this from a call to a return?

Reply:
When Connie and I were working on the error handling (therefore trying to trigger those catch zones in the `handleCreateList` function that calls this one), I noticed that the return is omitted in this one area. I'm pretty sure there should always be a return (and that's definitely the way I originally wrote it whoops), because we are expecting the function to return a Promise when we use it in `handleCreateList`. However, I didn't actually reproduce a "problem" caused by the lack of return (since it's not easy to trigger that area unless I specifically add some extra code to ensure duplicate tokens happen often). 

@yenly, @aprilleone or @Cjstoney: maybe one of you could look this over and provide some insight into whether the code could potentially break without that return?